bpdm-orchestrator/src/main/kotlin/org/eclipse/tractusx/bpdm/orchestrator/controller/CleaningTaskController.kt

```diff
@@ -52,6 +52,11 @@ class CleaningTaskController(
         }
     }
 
+    override fun resolveCleaningTasks(resultRequest: CleaningResultRequest) {
+        if (resultRequest.results.size > apiConfigProperties.upsertLimit)
+            throw BpdmUpsertLimitException(resultRequest.results.size, apiConfigProperties.upsertLimit)
+    }
```

Comment:
I suggest we could also check if at least result or any errors exist.
```suggestion
        resultRequest.results.forEach { entry -> 
            if (entry.result == null && entry.errors.isEmpty())
                throw Bpdm...
        }
```
And we could add a test for that.

Reply:
I added this validation logic but I was a bit torn about it as I think it could also be part of a separate validation service logic and not in the controller. We can still move it later if necessary